src/main/java/com/openlattice/hazelcast/StreamSerializerTypeIds.java

```diff
@@ -293,5 +292,17 @@
     REMOVE_ROLE_FROM_APP_PROCESSOR,
     ADD_ROLE_TO_APP_PROCESSOR,
     MATERIALIZED_VIEW_ACCOUNT,
-    ORGANIZATION_DATABASE
```

Comment:
we'll need a full, non-rolling restart for these deleted guys (although I think we might have needed that anyway? unsure)

Reply:
these were just moved during merge

Reply:
ah